core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -74,14 +74,25 @@ const showMarketDetails = async (web3, artifacts, emp) => {
       "Collateral pending/available to withdraw": fromWei(position.withdrawalRequestAmount.toString())
     });
 
-    actions = {
-      ...actions,
-      create: "Borrow more tokens",
-      redeem: "Repay tokens",
-      withdraw: "Withdraw collateral",
-      deposit: "Deposit collateral",
-      transfer: "Transfer position to new owner"
-    };
+    const hasPendingWithdrawal = position.requestPassTimestamp.toString() !== "0";
+    if (hasPendingWithdrawal) {
+      console.log(
+        "Because you have a pending withdrawal, other contract functions are blocked. Either execute or cancel your withdrawal."
+      );
+      actions = {
+        ...actions,
+        withdraw: "Withdraw collateral"
```

Comment:
Are we expecting to add a cancel withdrawal option to this list?

Reply:
Maybe we should rename this option to "Manage your withdrawals", because when you select this option, then you go to a deeper menu where you can cancel. Thoughts?

Reply:
I went ahead and did this, seems clearer to me.

Reply:
Ahhh, that makes sense. +1.